device/remove_current.go

```diff
@@ -0,0 +1,18 @@
+package device
```

Comment:
@BroderPeters You are here https://github.com/traggo/server/pull/14/commits/0253d94b0d642a4ae4ecbc1c24a2719854e08bd7